nfsv4/test/svg.c

```diff
@@ -0,0 +1,140 @@
+#include <runtime.h>
```

Comment:
Please confirm that we will be able to release this code under an open source license (probably MIT).

Reply:
yes, this is owned by me, but (a), its not complete and (b), i'm quite happy to have it released under mit